### PR TITLE
Simon dev

### PR DESCRIPTION
**Pull request by Slugger70**
Added a script to check toolshed versions and compare with our current ones.

### CODE REVIEW

File: toolshed_repo_updater/toolshed_repo_updater.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    else:
+        new_version = ts.repositories.get_ordered_installable_revisions(item['name'], item['owner'])[-1]
+
+    if old_version != new_version:
+        print "%s\t%s\t%s\t%s" % (item['name'], item['owner'], old_version, new_version)
+        item['revision'] = new_version
+
+    counter += 1
+    if counter == 20:
+        time.sleep(10)
```
Comment (nuwang) [inline toolshed_repo_updater/toolshed_repo_updater.py:67]: debug code?

File: toolshed_repo_updater/toolshed_repo_updater.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+print whole_yaml
+
+tool_list=whole_yaml['tools']
+
+print("Tool\tOwner\tCurrent\tLatest")
+counter = 0
+for item in tool_list:
+
+    tool_key = item['name']+'::'+item['owner']
```
Comment (nuwang) [inline toolshed_repo_updater/toolshed_repo_updater.py]: unused var?

File: toolshed_repo_updater/toolshed_repo_updater.py (hunk 1 above)
Comment (Slugger70) [inline toolshed_repo_updater/toolshed_repo_updater.py:67]: Are you asking if there is any or if the counter is debug. The counter is so I don't send too many requests to the REST api too quickly.. It tends to fail if I do that.

File: toolshed_repo_updater/toolshed_repo_updater.py (hunk 2 above)
Comment (Slugger70) [inline toolshed_repo_updater/toolshed_repo_updater.py]: Probably.

**Pull request merged**